_posts/2018-02-16-Stairway-To-Health-2.md

```diff
@@ -0,0 +1,344 @@
+---
+layout: post
+authors: [michael_vervloet, axel_bergmans]
+title: 'Stairway to Health 2.0 (the Ordina version)'
+image: /img/stairwaytohealth2/banner.jpg
+tags: [NodeJS, nestJs, MongoDB, Angular, ExpressJS, Express, TypeScript, Angular-CLI, Internet of Things, IoT, LoRa]
+category: IoT
+comments: true
+---
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/lightbox2/2.9.0/css/lightbox.css" />
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-grid-only@1.0.0/bootstrap.css" />
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/lightbox2/2.9.0/js/lightbox.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/bootstrap-grid-only@1.0.0/index.min.js"></script>
+
+
+<h2>Harder, Better, Faster, Stronger</h2>
+Here we are again, another blog post about [Stairway to Health](http://sth2-fe-prod-ordina-stairway-to-health.origin.ordina-jworks.io/dashboard).
+
+
+Why? Well, we've created our own Ordina version of the [Stairway to Health](http://sth2-fe-prod-ordina-stairway-to-health.origin.ordina-jworks.io/dashboard) application.
```

Comment:
It's down? http://sth2-fe-prod-ordina-stairway-to-health.origin.ordina-jworks.io/dashboard